ahoopes16/helianthus
# Mobile Friendly Page

# Summary
If accepted, this PR will:
- Make the Under Construction page mobile-friendly

# How to Test
1. `yarn && yarn start`
1. Visit http://localhost:3000 using Google Chrome
1. Go into the Developer Tools
1. Use the Device Emulator to check out different devices of varying sizes
1. Ensure that all sizes look good (within reason)
